### DISCUSSION TITLE
The responsive sidebar is not functioning properly.

### DISCUSSION THREAD

**Discussion by Tsuuuuuuun** [cotes2020/jekyll-theme-chirpy#1999] [Q&A]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Built from `jekyll-theme-chirpy`

### Description

After performing a git clone from GitHub and deploying it as is, the responsive sidebar is not functioning properly on mobile devices. I also ran `bundle exec jekyll serve` to check it locally, but it still doesn't work correctly. I have attached a video below.

https://github.com/user-attachments/assets/ac5618e9-ba7c-41f1-b5aa-b0c6ab61f5ae

### Operations you have already tried

I have looked through various files related to the sidebar, but since I am a beginner with HTML, CSS, and JavaScript, I couldn't fully understand them.

**Comment by kungfux** [comment]
Do you see any messages in the browser console?

I guess you may have issues with other functionality like search. If that's the case, then make sure to build javascript files:

```bash
$ npm install
$ npm run build
```

**Reply by Tsuuuuuuun** [reply]
Thank you for reply. I see in Microsoft edge and Safari.
Where should I execute this command? At `jekyll-theme-chirpy`?

**Reply by kungfux** [reply]
> Where should I execute this command? At jekyll-theme-chirpy?

Yes. Execute from bash or command line. Or, better use `tools/init.sh` script to initialize the environment.
Alternative option might be to use [Dev Containers](https://chirpy.cotes.page/posts/getting-started/#using-dev-containers-recommended-for-windows).

**Reply by Tsuuuuuuun** [reply]
I ran
```
$ npm install
$ npm run build
```
and it worked successfully! Thanks!
